bitnami/rabbitmq/Chart.yaml

```diff
@@ -23,4 +23,4 @@ name: rabbitmq
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/rabbitmq
   - https://www.rabbitmq.com
-version: 11.3.1
+version: 11.3.2
```

Comment:
As you are adding a new parameter, please bump the chart to a new minor version `11.4.0`